composer.json

```diff
@@ -31,55 +26,54 @@
   "repositories": [
     {
       "type": "composer",
-      "url": "https://wpackagist.org"
+      "url": "https://wpackagist.org",
+      "only": ["wpackagist-plugin/*", "wpackagist-theme/*"]
     }
   ],
   "require": {
     "php": ">=7.3",
-    "composer/installers": "^1.7",
-    "oscarotero/env": "^1.2.0",
+    "composer/installers": "^1.11",
+    "oscarotero/env": "^2.1",
     "pressbooks/pressbooks": "dev-dev",
     "pressbooks/pressbooks-aldine": "dev-dev",
     "pressbooks/pressbooks-book": "dev-dev",
     "pressbooks/pressbooks-clarke": "dev-dev",
     "pressbooks/pressbooks-donham": "dev-dev",
     "pressbooks/pressbooks-jacobs": "dev-dev",
     "rhubarbgroup/redis-cache": "^2.0",
+    "roots/bedrock-autoloader": "^1.0",
+    "roots/bedrock-disallow-indexing": "^2.0",
     "roots/multisite-url-fixer": "^1.1",
-    "roots/wordpress": "5.6.2",
+    "roots/wordpress": "5.8",
     "roots/wp-config": "1.0.0",
     "roots/wp-password-bcrypt": "1.0.0",
-    "vlucas/phpdotenv": "^3.4.0"
+    "vlucas/phpdotenv": "^5.3"
   },
   "require-dev": {
     "phpunit/phpunit": "^7",
-    "pressbooks/coding-standards": "^1.0",
+    "pressbooks/coding-standards": "dev-master",
+    "roave/security-advisories": "dev-master",
+    "squizlabs/php_codesniffer": "^3.5",
```

Comment:
Is also declared as a dependency in coding-standards master branch: https://github.com/pressbooks/coding-standards/blob/e332af77b246b088c5320934296746c1033a98cb/composer.json#L7. Might not be needed here as well?